From: "Emilio A. Escobar" <eescobar@gmail.com>
Date: Tue, 23 Oct 2012 19:35:48 -0400
Subject: sslstrip plugin links

cherry pick upstream commit 026ca28

Author: Emilio A. Escobar <eescobar@gmail.com>
Date:   Tue Oct 23 19:35:48 2012 -0400

Modified the target libraries for SSLStrip to remove dependency of X11 libraries such as GTK
---
 plug-ins/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plug-ins/CMakeLists.txt b/plug-ins/CMakeLists.txt
index 729dcf8..29ddd98 100644
--- a/plug-ins/CMakeLists.txt
+++ b/plug-ins/CMakeLists.txt
@@ -50,7 +50,7 @@ add_library(nbns_spoof MODULE nbns_spoof/nbns_spoof.c)
 set_target_properties(nbns_spoof PROPERTIES PREFIX "ec_")
 
 add_library(sslstrip MODULE sslstrip/sslstrip.c)
-target_link_libraries(sslstrip ${EC_LIBS})
+target_link_libraries(sslstrip ${FOUND_LIBCURL})
 set_target_properties(sslstrip PROPERTIES PREFIX "ec_")
 
 add_library(pptp_chapms1 MODULE pptp_chapms1/pptp_chapms1.c)
